compose/cli/main.py

```diff
@@ -269,13 +269,14 @@ def down(self, project, options):
         Usage: down [options]
 
         Options:
-            --rmi type      Remove images, type may be one of: 'all' to remove
-                            all images, or 'local' to remove only images that
-                            don't have an custom name set by the `image` field
-            -v, --volumes   Remove data volumes
+            --rmi type          Remove images, type may be one of: 'all' to remove
+                                all images, or 'local' to remove only images that
+                                don't have an custom name set by the `image` field
+            -v, --volumes       Remove data volumes
+            --remove-orphans    Remove orphaned containers
```

Comment:
I think `--remove-orphans` sounds better and it's one letter shorter. Grammatically I prefer it as well (i.e. "Remove orphaned what?")

I'll update the help text.